Resources/Prototypes/Entities/Structures/Doors/Windoors/windoor.yml

```diff
@@ -233,8 +265,8 @@
 
 - type: entity
   parent: WindoorSecure
-  id: WindoorSecureAtmosphericsLocked
-  suffix: Atmospherics, Locked
+  id: WindoorSecureVirologyLocked
+  suffix: Virology, Locked
```

Comment:
Virology access doesn't exist currently, and may never will. So it's probably best just to not include this and leave this for if/when it gets added.

Reply:
fair, i'll remove em'